self_enrollment_tool/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class SelfEnrollmentCourse(models.Model):
+
+    course_instance_id = models.IntegerField(unique=True)
+    role_id = models.IntegerField()
+    updated_by = models.CharField(max_length=15)
+    last_updated = models.DateTimeField(auto_now_add=True)
```

Comment:
Could we define the DB table name in the META class for this model?

    class Meta:
        db_table = 'self_enrollment_course'